packages/eds-core-react/src/components/TextField/TextField.tsx

```diff
@@ -29,7 +29,7 @@ type SharedTextFieldProps = {
   /** Variants */
   variant?: Variants
   /** Input unique id. This is required to ensure accesibility */
```

Comment:
Should update this comment which is used by storybook propstable. Perhaps just remove the entire comment